1-exercises/E-for-loop/exercise2.js

```diff
@@ -11,22 +11,18 @@
 */
 
 const WRITERS = [
-    "Virginia Woolf",
-    "Zadie Smith",
-    "Jane Austen",
-    "Bell Hooks",
-    "Yukiko Motoya"
-]
-
-const AGES = [
-    59,
-    40,
-    41,
-    63,
-    49
+  "Virginia Woolf",
+  "Zadie Smith",
+  "Jane Austen",
+  "Bell Hooks",
+  "Yukiko Motoya",
 ];
 
-// TODO - Write for loop code here
+const AGES = [59, 40, 41, 63, 49];
+
+for (let i = 0; i < WRITERS.length; i++) {
```

Comment:
nice! You can also use map when there is a transformation of an array into another array of the same size.